lib/devise/multi_email/models/authenticatable.rb

```diff
@@ -69,27 +52,18 @@ def find_first_by_auth_conditions(tainted_conditions, opts = {})
 
           if email && email.is_a?(String)
             conditions = filtered_conditions.to_h.merge(opts).
-              reverse_merge(emails: { email: email })
+              reverse_merge(multi_email_association.reflection.table_name => { email: email })
```

Comment:
> @allenwq the joins method takes the association name, but the where method needs to use the table name when using Hash conditions. Or am I looking at the wrong place?

I think where clause also takes the association name (not table name).  you can try on your local to confirm.

Actually our app uses `user_emails` as the table name and `emails` as the association name, the old implementation worked great before.